.golangci.yml

```diff
@@ -0,0 +1,196 @@
+# Copyright 2023 The Authors (see AUTHORS file)
```

Comment:
You shouldn't commit this unless this file deviates from the one in [abcyxz/pkg](https://github.com/abcxyz/pkg). If it does deviate from `pkg` then you'd need to specify it as a parameter on the call to the action. The ci workflow's action is configured to use the default which is in the `pkg` repo. Having this here will cause confusion in the future.

Reply:
Hmm the documentation for the workflow says:

The URL to a golangci file. This is only used if no file is found in the local directory.

Does that mean I don't need to specify? Otherwise it looks for a URL but I'm not sure what that would be since I'd want it to be whichever one is in the PR being run in case it's updated.